mne/cov.py

```diff
@@ -1713,6 +1718,8 @@ def _estimate_rank_meeg_signals(data, info, scalings, tol=1e-4,
         thresholded to determine the rank are also returned.
     """
     picks_list = _picks_by_type(info)
+    if scalings is None:
+        scalings = DEFAULTS['scalings']
```

Comment:
Pretty sure those two bits of code are not functionally equivalent. For example, with the current/old way, you can pass a partial list like `scalings=dict(eeg=10)` and it will automatically use the other default scalings values for the other keys, right?